src/Microsoft.AspNetCore.Mvc.Abstractions/ModelBinding/ModelBindingContext.cs

```diff
@@ -232,69 +82,93 @@ public ModelStateDictionary ModelState
         /// <remarks>
         /// The <see cref="ModelMetadata"/> property must be set to access this property.
         /// </remarks>
-        public Type ModelType => ModelMetadata?.ModelType;
+        public abstract Type ModelType { get; }
 
         /// <summary>
-        /// Gets or sets a model name which is explicitly set using an <see cref="IModelNameProvider"/>.
-        /// <see cref="Model"/>.
+        /// Represents the <see cref="ModelBinding.OperationBindingContext"/> associated with this context.
```

Comment:
Understood, but this is specifically a tactical perf PR. These lines of text didn't even change - they just moved.